From: Narendra_K@Dell.com <Narendra_K@Dell.com>
Subject: PCI: label: Fix compilation error when CONFIG_ACPI is unset
Patch-mainline: Yes
Git-commit: 07eefe1ca542129c24833c96ceb83b9aaa3b458d
References: bnc#697944, fate#311158
    
This patch fixes compilation error descibed below introduced by
the commit 6058989bad05b82e78baacce69ec14f27a11b5fd
    
drivers/pci/pci-label.c: In function ‘pci_create_firmware_label_files’:
drivers/pci/pci-label.c:366:2: error: implicit declaration of function ‘device_has_dsm’
    
Signed-off-by: Narendra K <narendra_k@dell.com>
Signed-off-by: Jesse Barnes <jbarnes@virtuousgeek.org>
Signed-off-by: Rafael J. Wysocki <rjw@suse.de>
---
 drivers/pci/pci-label.c |    6 ++++++
 1 file changed, 6 insertions(+)

Index: linux-2.6.32-SLE11-SP2/drivers/pci/pci-label.c
===================================================================
--- linux-2.6.32-SLE11-SP2.orig/drivers/pci/pci-label.c
+++ linux-2.6.32-SLE11-SP2/drivers/pci/pci-label.c
@@ -174,6 +174,12 @@ pci_remove_acpi_index_label_files(struct
 	return -1;
 }
 
+static inline bool
+device_has_dsm(struct device *dev)
+{
+	return false;
+}
+
 #else
 
 static const char device_label_dsm_uuid[] = {
